containers/base/Dockerfile

```diff
@@ -123,20 +124,6 @@ RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sour
     rm -rf /usr/share/i18n/locales/* && \
     cd /
 
-# Install ProtoBuf, TensorFlow
```

Comment:
That requires more changes. For example, datalab.mlalpha.* will need to be changed to add try catch in importing tensorflow. And the same thing when we import cloudml SDK since it references DataFlow.

What do we lose if we require TF and DataFlow from pydatalab? These are added to setup.py so that Jupyter+pydatalab works. We could do conditional import, but isn't the same can be said for almost all imports?